target/linux/ramips/image/Makefile

```diff
@@ -174,6 +174,11 @@ define Build/fonfxcimage
 	mv $@.new $@
 endef
 
+define Build/mitrastarimage
+	uimage_padhdr -l 160 -i $@ -o $@.new
+	mv $@.new $@
+endef
```

Comment:
Is there a specific reason why you add this to target Makefile and not to the subtarget-specific one?

Reply:
Mostly because that was where the other image fixup helpers were defined, including the Build/fonfxcimage which uses the same uimage_padhdr with a different pad length.  This is only used by one MT7620 device.

I considered changing Build/fonfxcimage to a generic padder with length as an option, but ended up with separate macros for readability.

Anyway, I certainly see your point.  Moved this into mt7621.mk.